TestAssets/TestProjects/DotNetCoreTestProject/project.json

```diff
@@ -0,0 +1,21 @@
+{
+  "dependencies": {
+    "MSTest.TestAdapter": "1.0.3-preview",
```

Comment:
It doesn't updated build by build. We release new version on some time interval after fixing bugs or adding new feature in it.